Update bug report template (#395)
OpenDevin/OpenDevin: .github/ISSUE_TEMPLATE/bug_report.md

assignees: ''

---
<!-- You MUST fill out this template. We will close issues that don't include enough information to reproduce -->
#### Describe the bug
<!-- a short description of the problem -->

#### Setup and configuration
**Current version**:
<!-- run `git log -n 1` to see this -->
```bash
```

<!-- tell us everything about your environment -->
**My config.toml and environment vars** (be sure to redact API keys):
```toml
```

> **My model and agent** (see top-right of the UI):

Author: ```suggestion
**My model and agent** (see the settings in the UI):
```
